app.js

```diff
@@ -1,9 +1,19 @@
 const express = require("express");
 const apiRouter = require("./routes/api-router");
-
+const result = dotenv.config();
 const app = express();
 app.use(express.json());
 
+if (result.error) {
+  throw result.error;
```

Comment:
You're throwing an error here at the top level of the Node.js app... there's nothing that can catch this error, and the result is  the app crashing. When this happens - _someone_ will have go in and restart the app.

Try and find a more elegant way to report the error, and not kill the app.